modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractTypeScriptClientCodegen.java

```diff
@@ -174,6 +175,11 @@ public String toModelName(String name) {
                 return modelName;
             }
 
+            if(languageSpecificPrimitives.contains(name)){
```

Comment:
Please add a space before and after the condition: https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style